modules/apps/bean-portlet/bean-portlet-cdi-extension-sample-test/pom.xml

```diff
@@ -0,0 +1,69 @@
+<?xml version="1.0"?>
+
+<project
+	xmlns="http://maven.apache.org/POM/4.0.0"
+	xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+	xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/maven-v4_0_0.xsd"
+>
+	<modelVersion>4.0.0</modelVersion>
+
+	<!-- Note: The parent POM contains "thin" and "cdi" profiles -->
+
+	<parent>
+		<groupId>com.liferay.faces.archetype</groupId>
+		<artifactId>com.liferay.faces.archetype.parent</artifactId>
+		<version>6.1.1</version>
+	</parent>
+	<groupId>sample.bean.portlet</groupId>
+	<artifactId>sample.bean.portlet</artifactId>
+	<packaging>war</packaging>
+	<name>Sample CDI Bean Project</name>
+	<version>1.0.0</version>
+	<build>
```

Comment:
@Kvale1733, due to `<artifactId>sample.bean.portlet</artifactId>`, the servletContextName will `sample.bean.portlet`. This caused `liferay-plugin-package.properties` won't be read. You may refer to PluginPackageUtil#_readPluginPackageServletContext().

So please rename and refer to the below:
```
<groupId>com.liferay</groupId>
<artifactId>sample-bean-portlet</artifactId>
```